deploy/kubernetes/overlays/prow-gke-release-staging-head/controller_add_snapshotter.yaml

```diff
@@ -0,0 +1,17 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-snapshotter
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-snapshotter:2.1-canary
```

Comment:
We want "canary" here to test latest changes in the master branch

Reply:
Done